front-end/src/ui-components/Address.tsx

```diff
@@ -58,20 +72,30 @@ export default styled(Address)`
 	align-items: center;
 
 	.content {
-		padding-left: 1rem;
 		display: inline-block;
 		line-height: 1.6rem;
 	}
 
+	.identicon {
+		margin-right: 1rem;
+	}
+
 	.header {
 		color: black_text;
 		font-weight: 500;
 		font-size: sm;
 		filter: grayscale(100%);
+		margin-right: 0.4rem;
 	}
 
 	.description {
 		color: grey_primary;
 		font-size: xs;
+		margin-right: 0.4rem;
+	}
+
+	&.inline, .inline {
```

Comment:
Is there a reason to double this one? one or the other should be sufficient I imagine.